src/main/kotlin/pl/edu/uj/ii/ksi/mordor/persistence/managers/FileManager.kt

```diff
@@ -0,0 +1,18 @@
+package pl.edu.uj.ii.ksi.mordor.persistence.managers
+
+import java.io.File
+import pl.edu.uj.ii.ksi.mordor.persistence.entities.FileEntry
+
+interface FileManager {
```

Comment:
How does this class relates to RepositoryService?

Reply:
This class was supposed to be facade of FileEntry related services.

Reply:
since FileEntries are secondary to actual files it looks like you are duplicating the functionality of RepositoryService.